package.json

```diff
@@ -1,36 +1,36 @@
 {
-  "name": "angular-waypoints",
-  "version": "2.0.0",
-  "description": "An AngularJS module for working with Waypoints",
-  "main": "dist/angular-waypoints.js",
-  "scripts": {
-    "test": "grunt test"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/zumba/angular-waypoints.git"
-  },
-  "keywords": [
-    "angular",
-    "waypoints",
-    "scroll"
-  ],
-  "author": "Stephen Young <stephen.young@zumba.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/zumba/angular-waypoints/issues"
-  },
-  "homepage": "https://github.com/zumba/angular-waypoints",
-  "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-concat": "^0.5.0",
-    "grunt-contrib-jasmine": "^0.7.0",
-    "grunt-contrib-jshint": "^0.10.0",
-    "grunt-contrib-uglify": "^0.5.1",
-    "grunt-wrap": "^0.3.0",
-    "install": "^0.1.7",
-    "lodash": "^2.4.1",
-    "npm": "^1.4.23"
-  }
+	"name": "angular-waypoints",
+	"version": "2.0.0",
+	"description": "An AngularJS module for working with Waypoints",
+	"keywords": [
+		"angular",
+		"waypoints",
+		"scroll"
+	],
+	"homepage": "https://github.com/zumba/angular-waypoints",
+	"bugs": {
+		"url": "https://github.com/zumba/angular-waypoints/issues"
+	},
+	"license": "MIT",
+	"author": "Stephen Young <stephen.young@zumba.com>",
+	"main": "dist/angular-waypoints.js",
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/zumba/angular-waypoints.git"
+	},
+	"scripts": {
+		"test": "grunt test"
+	},
+	"devDependencies": {
+		"grunt": "^0.4.5",
+		"grunt-contrib-clean": "^0.6.0",
+		"grunt-contrib-concat": "^0.5.0",
+		"grunt-contrib-jasmine": "^0.7.0",
+		"grunt-contrib-jshint": "^0.10.0",
+		"grunt-contrib-uglify": "^0.5.1",
+		"grunt-wrap": "^0.3.0",
+		"install": "^0.1.7",
+		"lodash": "^2.4.1",
+		"npm": "^1.4.23"
+	}
```

Comment:
@earlyster Could you remove the whitespace changes to this file?  They make it hard to tell what, if anything, actually changed in this file.